modules/core/src/main/java/org/locationtech/jts/geom/CoordinateSequence.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Vivid Solutions.
+ * Copyright (c) 2018 Vivid Solutions, and others.
```

Comment:
I don't think this is a valid copyright?

Reply:
It is the format used by eclipse source code so I figured it was good:


![](http://www.codejava.net/images/articles/ides/eclipse/copyright/Copyright_Tool_default_setting.png)